lib/native/mqtt.spec.ts

```diff
@@ -221,7 +218,7 @@ test('MQTT Websocket', async () => {
 
 test('MQTT Pub/Sub', async () => {
 
-    io.enable_logging(io.LogLevel.TRACE);
+    //io.enable_logging(io.LogLevel.TRACE);
```

Comment:
you want to remove this after testing?

Reply:
I'll delete it, it's left from some testing a while back.